File: Contracts/Model/Enterprise/QuoteUpdateDto.cs
namespace Contracts.Model.Enterprise;

public class QuoteUpdateDto
{
    public string ContactNumber { get; set; }
    public string? ExtraInfo { get; set; }
    public string? Email { get; set; }
    public string EventName { get; set; }
    public DateTimeOffset EventDate { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
    public string Location { get; set; }
    public ICollection<QuoteItemDto>? QuoteItems { get; set; }
}

File: ClientApp/src/app/interfaces/enterprise/quoteUpdateDto.ts
import { QuoteItemDto } from "./quoteItemDto";

export interface QuoteUpdateDto {
    contactNumber: string;
    extraInfo: string | null;
    email: string | null;
    eventName: string;
    eventDate: string;
    createdAt: string;
    location: string;
    quoteItems: QuoteItemDto[] | null;
}